app.html

```diff
@@ -558,16 +560,37 @@
 
 function SaveFont() {
   var font_select = document.getElementById('font_select');
-  console.log("SAVE FONT " + font_select.value);
-  if (presets[current_preset_num].FONT != font_select.value) {
-    presets[current_preset_num].FONT = font_select.value;
-    Send("set_font " + font_select.value);
+  var addon_select = document.getElementById('font_select2');
+  var font_path = font_select.value;
+  if (addon_select.value != "none"){
+    font_path += ";" + addon_select.value;
+  }
+  console.log("SAVE FONT " + font_path);
+  if (presets[current_preset_num].FONT != font_path) {
+    presets[current_preset_num].FONT = font_path;
+    Send("set_font " + font_path);
   }
 }
 
 function make_font_field(value) {
-  ret="<div class=edit-select><span class=title>font<br><select class=myselect id=font_select name=font_select onchange='SaveFont()'>";
-  ret+= make_select(font_lines, value);
+  var tmp = value.split(";");
+  var font_ = value;
+  var addon_ = "none"
+  if (tmp.length > 1){
```

Comment:
What happens if there are more than 2 components in the font search path?

Reply:
You got me there. I was still debating how to solve that one. I could imagine an "+" sign in the edit box to add additional folders. After clicking the "+" the pull down menu will show  to select the additional folder. The "+" will shift down allowing you to add as many search folder as you want. Nested folders within other soundfonts would be challenging, unless we allow for a text-input edit field to specify the folders you want to include (thsi is mor prone to erroneous entries)
Doing so would also solve the necessity to have a "none" showing currently in the additional folder if you don't have a common folder for any of your sounds. 
For now most likely is that you have your menu sounds in a common folder. With the previous version of WebAPP you would loose you common folder, by the way the pulldown only selects a single sound font.

Something for the ToDo list...

Reply:
I think it's fine to only have two selectboxes, but it should always be possible to select what was already there, and perhaps also what was listed in other presets.  So like if the path is foo;bar;gazonk, maybe the top selectbox would say "foo" and the bottom one would say "bar;gazonk" ?  Or, we could just have "foo;bar;gazonk" in the top box and None in the bottom. Either way, I don't think it's acceptable for the "gazonk" to just disappear.

Reply:
In the current version the check box indeed shows the whole path (although corrupted because of the starjedi font not showing ";" correctly. As soon as you alter the font, it will overwrite with the full path, with only the selected font.
Having the entries of the other presets selectable would be a nice idea to keep the full path with multiple search folders. 
Is there a way to get the star-jedi to show ";" correctly?

Reply:
I don't think StarJedi has ; unfortunately.
Should be possible to switch in a font just for that character though.
I'm ok with switching fonts changing the number of font elements, as long as you can switch back too.